### DISCUSSION TITLE
Part2 CH01_01 강의가 실행이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by snow-shlim** [tedilabs/fastcampus-devops#18] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/64842804/138026227-0e1ba146-23a6-4a7c-8862-638311187e5d.png)

Part 2. AWS 기반 소규모 & 중규모 아키텍트 설계
CH01_01. (개요) 클라우드 컴퓨팅

해당 강의가 불러와지질 않습니다.

확인 부탁드립니다.

**Comment by posquit0** [comment, marked as answer]
패스트캠퍼스에 문의 부탁드립니다! ㅠ.ㅠ
여기는 박병진 강사 강의 내용에 대한 질문 응답 공간입니다. :)

**Reply by snow-shlim** [reply]
죄송합니다. 패스트캠퍼스에서 글 올리는 곳을  못 찾아서 여기에 올렸습니다.
질문 삭제해 주세요
